model/edgeless.ml

```diff
@@ -140,9 +147,11 @@ module Controller : sig
 
   val start : t -> Workflow.t -> t * wid
   val stop : t -> wid -> t
+  val list : t -> t * (wid * Orchestrator.t list)
```

Comment:
`list` is a bit generic -- perhaps having two methods would be better (if I understand the type `t` definition below correctly):
```ocaml
val orchestrators  : t -> Orchestrators.t list
val workflows : t -> (wid * Orchestrator.t list) Hashtbl.t
val workflow : t -> wid -> Orchestrator.t list
```
?